bundle/src/main/java/com/jfrog/ide/eclipse/utils/ProjectsMap.java

```diff
@@ -4,27 +4,28 @@
 import java.util.TreeMap;
 
 import org.apache.commons.lang3.StringUtils;
-import org.jfrog.build.extractor.scan.DependenciesTree;
+import org.jfrog.build.extractor.scan.DependencyTree;
 import org.jfrog.build.extractor.scan.GeneralInfo;
 
+
 /**
  * Map of projects. Used in the UI to show the projects sorted by project name
  * and path.
  * 
  * @author yahavi
  */
-public class ProjectsMap extends TreeMap<ProjectsMap.ProjectKey, DependenciesTree> {
+public class ProjectsMap extends TreeMap<ProjectsMap.ProjectKey, DependencyTree> {
 	private static final long serialVersionUID = 1L;
 
 	/**
 	 * Put a project in the map.
 	 * 
 	 * @param projectName      - The project name.
-	 * @param dependenciesTree - The dependencies tree.
-	 * @return dependenciesTree.
+	 * @param dependencyTree - The dependencies tree.
+	 * @return dependencyTree.
 	 */
-	public DependenciesTree put(String projectName, DependenciesTree dependenciesTree) {
-		return super.put(createKey(projectName, dependenciesTree.getGeneralInfo()), dependenciesTree);
+	public DependencyTree put(String projectName, DependencyTree dependencyTree) {
```

Comment:
I got to ask - why is the rename between `DependenciesTree` to `DependencyTree` ?

Reply:
This object is implemented in the dependency project `build-info`. The name of this object was changed in that project, therefor we had to refactor the usage of it in our plugin.